examples/next/pages/_app.page.tsx

```diff
@@ -3,13 +3,15 @@
 // https://nextjs.org/docs/advanced-features/custom-app
 import App from 'next/app';
 import { Amplify } from 'aws-amplify';
+import * as Auth from 'aws-amplify/auth';
 import { Hub } from 'aws-amplify/utils';
 import { Authenticator, AmplifyProvider } from '@aws-amplify/ui-react';
 import { useEffect } from 'react';
 import { MapProvider, useMap } from 'react-map-gl';
 
 if (typeof window !== 'undefined') {
   window['Amplify'] = Amplify;
+  window['Auth'] = Auth;
```

Comment:
Am a bit concerned that we need this now but not with v5. Mostly that maybe it might be covering up an underlying issue

Reply:
Oh that's because previously everything was imported from Amplify. Either way this doesn't really fix much, just fixes some build errors in e2e tests but the tests themselves still fail lol

Reply:
MSTM 😄